_targets.R

```diff
@@ -63,7 +63,8 @@ min_windows <- 10  ##Must have this many windows available in order to plot
 
```

Comment:
at the top of the `_targets.R` script, add a `dir.create` for `1_fetch/out/logs/`